products/compute/api.yaml

```diff
@@ -1285,6 +1285,67 @@ objects:
           An optional textual description of the valid disk size, such as
           "10GB-10TB".
         output: true
+  - !ruby/object:Api::Resource
+    name: 'DiskResourcePolicyAttachment'
+    input: true
+    base_url: projects/{{project}}/zones/{{zone}}/disks/{{disk}}
+    create_verb: :POST
+    create_url: projects/{{project}}/zones/{{zone}}/disks/{{disk}}/addResourcePolicies
+    delete_verb: :POST
+    delete_url: projects/{{project}}/zones/{{zone}}/disks/{{disk}}/removeResourcePolicies
+    self_link: projects/{{project}}/zones/{{zone}}/disks/{{disk}}
+    nested_query: !ruby/object:Api::Resource::NestedQuery
+      keys:
+        - resourcePolicies
+      is_list_of_ids: true
+    identity:
+      - name
+    description: |
+      Disk resource policies define a schedule for taking snapshots and a
+      retention period for these snapshots.
+    async: !ruby/object:Api::Async
+      operation: !ruby/object:Api::Async::Operation
+        kind: 'compute#operation'
+        path: 'name'
+        base_url: 'projects/{{project}}/zones/{{zone}}/operations/{{op_id}}'
+        wait_ms: 1000
+        timeouts: !ruby/object:Api::Timeouts
```

Comment:
Looks like an x-merge with https://github.com/GoogleCloudPlatform/magic-modules/pull/2283 caused this to generate invalid code, previously this block was ignored. https://github.com/GoogleCloudPlatform/magic-modules/pull/2325 fixes default values, so you can merge that to fix this.

Alternatively, it's probably a bit faster to work around it- you can add `delete_minutes` here as well. I'd define both of these as `4` minutes in that case, we have no strong reason to go lower than the default.